activerecord-pluck_in_batches.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", ">= 4.2.7", "< 5.2.0"
+  spec.add_dependency "activerecord", ">= 4.2.7", "< 6.0.0"
```

Comment:
Not a blocker by any means, but any reason to continue to exclude Rails 6.x?

Reply:
Haven't tried it... 🤷‍♂ PRs are cheap. 😆

Reply:
Good enough for me 😆